src/components/Apps/AppDetail/InstallAppModal/InstallAppForm.tsx

```diff
@@ -13,7 +14,7 @@ interface IInstallAppFormProps {
   namespaceError: string;
   valuesYAMLError: string;
   version: string;
-  availableVersions: IAppCatalogAppVersion[];
+  availableVersions: IVersion[];
```

Comment:
Moved the mapping outside the component, so it would be easier to pass the required data structure right away